source/includes/posts/_02-project-design-preparation.md

```diff
@@ -451,19 +451,19 @@ Typically satellite images that have been purchased from a major provider like D
 
 Imagery can be purchased from companies such as DigitalGlobe and GeoEye, but this can quickly become expensive. The cost per square kilometer is around $15. It is usually best to find freely available sources of imagery, which are also more sustainable in the long run.
 
-In some cases, imagery is made available through the U.S. Department of State’s Imagery to the Crowd initiative. This is typically provided in the aftermath of disasters to aid remote mappers. Learn more about Imagery to the Crowd at <b>mapgive.state.gov</b>. Remember, too, that using aerial imagery as a background layer is generally considered to be the easiest way to map, particularly when many buildings need to be digitized.
+In some cases, imagery is made available through the U.S. Department of State’s Imagery to the Crowd initiative. This is typically provided in the aftermath of disasters to aid remote mappers. Learn more about Imagery to the Crowd at <b>[mapgive.state.gov](mapgive.state.gov)</b>. Remember, too, that using aerial imagery as a background layer is generally considered to be the easiest way to map, particularly when many buildings need to be digitized.
 
 Figure 2.8: A Field Papers Atlas Map, without Imagery, Used in Sumba Barat, Indonesia
 <div class="c-box-image">
     <img src="/images/posts/2/2-8.tif" alt="figure 2.8">
   </div>
-  <em>Source: Field Papers (http://www.fieldpapers.org).</em>
+  <em>Source: Field Papers [fieldpapers.org](http://www.fieldpapers.org)</em>
 
 Figure 2.9: A Field Paper Atlas Map Example
 <div class="c-box-image">
     <img src="/images/posts/2/2-9.tif" alt="figure 2.9">
   </div>
-  <em>Source: Field Papers (http://www.fieldpapers.org).</em>
+  <em>Source: Field Papers [fieldpapers.org](http://www.fieldpapers.org)</em>
```

Comment:
The text between brackets "[ ]" is the link name. In this case I think it's "Field Papers", if not you are going to get: _Field Papers_ [fieldpapers.org](http://www.fieldpapers.org). Otherwise changing it would be: [Field Papers](http://www.fieldpapers.org)